app/models/relationship.rb

```diff
@@ -1,4 +1,6 @@
 class Relationship < ApplicationRecord
+
   belongs_to :leader, class_name: 'User'
   belongs_to :follower, class_name: 'User'
+
```

Comment:
Just a suggestion - To add
validates :leader_id, presence: true
validates :follower, presence: true 

can simply RSpec test later.

Reply:
👍 okie!